### PR TITLE
add support for the cmk EnableKey | DisableKey  restful API

### PR DESCRIPTION
**Pull request by panpan0721**
this restful API will be exposed to public as:

 - POST <ehsm_srv_address>/ehsm?Action=EnableKey
 - POST <ehsm_srv_address>/ehsm?Action=DisableKey

add the corresponding test cases.

test: passed the unittest

Signed-off-by: panpan0721 <2271409777@qq.com>

### CODE REVIEW

File: ehsm_kms_service/key_management_apis.js [hunk 1]
```diff
@@ -82,8 +96,68 @@ const deleteKey = (payload, res, DB) => {
     })
 }
 
+const enableKey = (payload, res, DB) => {
+  const query = {
+    selector: {
+      _id: `cmk:${payload.keyid}`,
+    },
```
Comment (yang8621) [inline ehsm_kms_service/key_management_apis.js:103]: is appid/creator checked?

File: ehsm_kms_service/key_management_apis.js [hunk 2]
```diff
@@ -82,8 +96,68 @@ const deleteKey = (payload, res, DB) => {
...
+    })
+    .catch((err) => {
+      res.send(_result(400, 'enableKey failed', { err }))
+    })
+}
+
+const disableKey = (payload, res, DB) => {
+  const query = {
+    selector: {
+      _id: `cmk:${payload.keyid}`,
```
Comment (yang8621) [inline ehsm_kms_service/key_management_apis.js:130]: is appid/creator checked?

File: ehsm_kms_service/key_management_apis.js (hunk 1 above)
Comment (wanghouqi) [reply ehsm_kms_service/key_management_apis.js:103]: It will be fix.

File: ehsm_kms_service/key_management_apis.js (hunk 2 above)
Comment (wanghouqi) [reply ehsm_kms_service/key_management_apis.js:130]: It will be fix.

File: ehsm_kms_service/router.js [hunk 3]
```diff
@@ -180,6 +192,12 @@ const router = async (p) => {
     case key_management_apis.DeleteAllKey:
       deleteALLKey(appid, res, DB)
       break
+    case key_management_apis.EnableKey:
+      enableKey(appid, payload, res, DB)
```
Comment (yang8621) [inline ehsm_kms_service/router.js:196]: Could you also check other funtions e.g. @line 189:
    case key_management_apis.DeleteKey:
      deleteKey(payload, res, DB)
      break

Is appid also required?

Comment (yang8621) [reply ehsm_kms_service/router.js:196]: Please submit a new patch to fix the issue in DeleteKey.

**Review by yang8621 — APPROVED** [review]

File: ehsm_kms_service/router.js (hunk 3 above)
Comment (wanghouqi) [reply ehsm_kms_service/router.js:196]: Ok,I will fix it.

**Review by syan10 — APPROVED** [review]

**Pull request merged**